### PR TITLE
use .case_id instead of .id

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?186404
@czue 

cc @biyeun 
FYI @benrudolph

### CODE REVIEW

**Comment by snopoke** [comment]
Hard to know for sure but I think I got all the instances of `case.id`

File: corehq/apps/hqcase/tests/test_case_assigment.py [hunk 1]
```diff
@@ -36,39 +36,39 @@ def test_assign_to_user_in_different_domain(self):
...
-            assign_case(case.id, other_user._id)
+            assign_case(case.case_id, other_user._id)
             self.fail('reassigning to user in wrong domain should fail')
         except CaseAssignmentError:
             pass
 
     def test_assign_no_related(self):
         self._make_tree()
-        assign_case(self.primary.id, self.primary_user._id, include_subcases=False, include_parent_cases=False)
+        assign_case(self.primary.case_id, self.primary_user._id, include_subcases=False, include_parent_cases=False)
```
Comment (dimagimon) [inline corehq/apps/hqcase/tests/test_case_assigment.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/hqcase/tests/test_case_assigment.py [hunk 2]
```diff
@@ -77,13 +77,13 @@ def test_assign_to_group(self):
...
         self._make_tree()
-        assign_case(self.primary.id, group._id, include_subcases=True, include_parent_cases=True)
+        assign_case(self.primary.case_id, group._id, include_subcases=True, include_parent_cases=True)
         self._check_state(new_owner_id=group._id, expected_changed=self.all)
 
     def test_assign_noop(self):
         self._make_tree()
         num_forms = get_number_of_forms_in_all_domains()
-        res = assign_case(self.primary.id, self.original_owner._id, include_subcases=True, include_parent_cases=True)
+        res = assign_case(self.primary.case_id, self.original_owner._id, include_subcases=True, include_parent_cases=True)
```
Comment (dimagimon) [inline corehq/apps/hqcase/tests/test_case_assigment.py]: E501 line too long (122 > 115 characters)

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by czue** [comment]
thanks!

**Comment by benrudolph** [comment]
ah shoot sorry about that didn't realize that could happen

**Comment by benrudolph** [comment]
i don't see `case_id` in this list: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static/app_manager/json/case-reserved-words.json#L22. what makes sure that that won't happen again?

**Comment by benrudolph** [comment]
nvm, this is something we set ourselves and why it doesn't have to be an `@property`

**Comment by czue** [comment]
that's weird, it definitely seems like it should be there. @dannyroberts might know a reason, but my guess is that it's an oversight

**Comment by dannyroberts** [comment]
Yeah it should definitely be in that list.

**Comment by snopoke** [comment]
https://github.com/dimagi/commcare-hq/pull/8969

**Pull request merged**